src/main/webapp/app/shared/link-preview/components/link-preview-container/link-preview-container.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="!multiple && linkPreviewService.links.length > 0; then first; else list"></ng-container>
+
+<ng-template #first>
+    <jhi-link-preview [linkPreview]="linkPreviews[0]" [showLoadingsProgress]="showLoadingsProgress" [hasError]="hasError" [loaded]="loaded"> </jhi-link-preview>
+</ng-template>
+<ng-template #list>
+    <div *ngFor="let preview of linkPreviews; trackBy: trackLinks">
+        <jhi-link-preview [linkPreview]="preview" [showLoadingsProgress]="showLoadingsProgress" [hasError]="hasError" [loaded]="loaded"> </jhi-link-preview>
+    </div>
+</ng-template>
```

Comment:
i had this `multiple ` field and this way, by using separate templates for the single and multiple link preview cases, the code becomes more readable. But since i remove the field now i will use the list only.